js/app.js

```diff
@@ -118,8 +118,9 @@
    * using the Function prototype method: call
    * to set the context to the correct object (the current context)
    */
-
-
+   btn6.addEventListener("click", function(){
+    setMyLightGreen.call(btn6);
```

Comment:
This one is asking for the current context. Replace 'btn6' with the current context